src/storekit.cs

```diff
@@ -659,6 +659,10 @@ interface SKStoreProductParameterKey
 		[MacCatalyst (14,0)]
 		[Field ("SKStoreProductParameterAdNetworkVersion")]
 		NSString AdNetworkVersion { get; }
+		
+		[Mac (12, 0), iOS (15, 0), MacCatalyst (15, 0)]
```

Comment:
that needs a `[NoTV]` if it's not available on tvOS
otherwise it will default to the minimum value (9.0) since it will fallback to the type definition (which is supported on tvOS)

There's no need for a `[NoWatch]` here since it's already present at the type-level

Reply:
header files don't say anything about TV, but the [web docs](https://developer.apple.com/documentation/storekit/skstoreproductparametercustomproductpageidentifier?changes=_4&language=objc) indicate availability, so I'm adding it in!